Formula/a/azure-pim-cli.rb

```diff
@@ -0,0 +1,24 @@
+class AzurePimCli < Formula
+  desc "Unofficial CLI for Azure Privileged Identity Management (PIM) roles"
+  homepage "https://github.com/demoray/azure-pim-cli"
+  url "https://github.com/demoray/azure-pim-cli/archive/refs/tags/0.6.1.tar.gz"
+  sha256 "04f8157a5645d2c6e388b4bd151effda7545779c79dc537a2dd8dcba4d14379f"
+  license "MIT"
+  head "https://github.com/demoray/azure-pim-cli.git", branch: "main"
+
+  depends_on "pkg-config" => :build
+  depends_on "rust" => :build
+  depends_on "azure-cli"
+
+  on_linux do
+    depends_on "openssl@3"
+  end
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/az-pim --version")
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula